tests/integration_tests/data_steward/utils/participant_summary_requests_test.py

```diff
@@ -0,0 +1,185 @@
+"""
+Integration Test for the deactivated_participants module
+
+Ensures that get_token function fetches the access token properly, get_deactivated_participants
+    fetches all deactivated participants information, and store_participant_data properly stores all
+    the fetched deactivated participant data
+
+Original Issues: DC-797, DC-971 (sub-task), DC-972 (sub-task)
+
+The intent of this module is to check that GCR access token is generated properly, the list of
+    deactivated participants returned contains `participantID`, `suspensionStatus`, and `suspensionTime`,
+    and that the fetched deactivated participants data is stored properly in a BigQuery dataset.
+"""
+
+# Python imports
+import unittest
+import mock
+
+# Third party imports
+import pandas
+import pandas.testing
+import google.auth.transport.requests as req
+from google.auth import default
+
+# Project imports
+import utils.participant_summary_requests as psr
+import app_identity
+from utils import auth
+
+
+class ParticipantSummaryRequests(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+    def setUp(self):
+        self.project_id = app_identity.get_application_id()
+        self.dataset_id = 'pipeline_tables'
```

Comment:
@dcampbell-vumc You've parameterized `dataset_id` in your code.  You could make this any dataset you choose.  Unless I've missed a spot.  Then you could delete the entirety of the dataset without worrying about removing someone else's test code.  Actually, you should change this hard coded name.  This will cause collisions between branches.

Reply:
I normally would have used `fake_dataset` for the dataset_id however, I thought having it named that would cause collisions since several integration tests use `fake_dataset`. I probably am misremembering though...

Reply:
The problem exists either way, both are hard coded names.  You need to use a parameterized variable for this.  The cleaning rules set the dataset id by doing something like this:
```suggestion
        dataset_id = os.environ.get('COMBINED_DATASET_ID')
```
This uses the available environment variables.  These are set in the circleci config as a combination of your user name and branch name.  This means they will be unique.

Reply:
Ah, I understand now!